app/routes/missions/swift.md

```diff
@@ -25,39 +25,52 @@ https://swift.gsfc.nasa.gov/archive/
+| `SWIFT_XRT_SPECTRUM`      | 40–81 s                     | N/A                | Spectrum (few-to-100 s integration)                         |
+| `SWIFT_XRT_LC`            | 141–910 s                   | N/A                | The x-ray lightcurve (106–826 s)                            |
+| `SWIFT_UVOT_FCHART`       | ~260 s                      | N/A                | Thresholded pixel clusters                                  |
+| `SWIFT_UVOT_DBURST`       | ~320 s                      | N/A                | All pixels from an 80×80 pixel subarray                     |
+| `SWIFT_UVOT_POS`          | 1–3 h                       | &lt;2″             | UVOT afterglow location (Gnd-generated only)                |
+| `SWIFT_UVOT_POS_NACK`     | 1–3 h                       | N/A                | Could not find an UVOT afterglow location                   |
+| **Ground Processed:**     |                             |                    |                                                             |
+| `SWIFT_BAT_GRB_LC_PROC`   | ~225 s                      | 1–3′               | Lightcurve (also has the position)                          |
+| `SWIFT_XRT_IMAGE_PROC`    | 40–70 s                     | &lt;6″             | 2′×2′ FOV (~10 s integration)                               |
+| `SWIFT_XRT_SPECTRUM_PROC` | 50–90 s                     | N/A                | The spectrum from that integration (~10 s)                  |
+| `SWIFT_UVOT_FCHART_PROC`  | ~280 s                      | N/A                | Thresholded pixel clusters                                  |
+| `SWIFT_UVOT_DBURST_PROC`  | ~340 s                      | N/A                | All pixels from an 80×80 pixel subarray                     |
+| **Special:**              |                             |                    |                                                             |
+| `SWIFT_BAT_SLEW_POS`      | 1–6 h                       | 1–7′               | Bursts & Transient found in the slew data (BATSS)           |
+| `SWIFT_BAT_MONITOR`       | 1–12 h                      | 1–7′               | Flares from known sources                                   |
+| `SWIFT_BAT_SUBSUB`        | 1–12 h                      | 1–7′               | Sub-Sub-Threshold blips in all image produced on-board      |
+| `SWIFT_BAT_KNOWN_SRC`     | 1–12 h                      | 1–7′               | Peaks of known-sources found in all image produced on-board |
+
+**Swift GRB Annual Trigger Detection Rates:**
+
+| Instrument   | Long GRBs | Short GRBs | Description                   |
+| ------------ | --------- | ---------- | ----------------------------- |
+| `SWIFT_BAT`  | 70-80     | 7-9        | On-board trigger rate         |
+| `SWIFT_XRT`  | 60-65     | 40-50      | XRT follow-up detections      |
+| `SWIFT_UVOT` | 20-25     | 0-1        | UVOT follow-up detections     |
+| `SWIFT_BAT`  | 8-10      | 2-4        | GUANO arcminute recovery rate |
```

Comment:
No need to typeset instrument names in `monospace` as if they were code.

Also, use en dashes (–, option+dash on macOS) for numerical ranges.

```suggestion
| BAT | 70–80     | 7–9        | On-board trigger rate         |
| XRT  | 60–65     | 40–50      | XRT follow-up detections      |
| UVOT | 20–25     | 0–1        | UVOT follow-up detections     |
| `SWIFT_BAT`  | 8–10      | 2–4        | GUANO arcminute recovery rate |
```